gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GoogleHadoopFileSystemBase.java

```diff
@@ -966,6 +1005,8 @@ public boolean mkdirs(Path hadoopPath, FsPermission permission) throws IOExcepti
 
     URI gcsPath = getGcsPath(hadoopPath);
     try {
+      // Update the statistics of mkdirs
+      entryPoint(GhfsStatistic.INVOCATION_MKDIRS);
```

Comment:
Should we have entry point at line 1002 to ensure we capture validation failures as invocations as well ?

Same goes for all methods

Reply:
Updated